templates/cloud/index.html

```diff
@@ -386,6 +386,7 @@ <h2>Landscape</h2>
   <div class="row">
     <p id="fn1" class="note"><a href="#r1">*</a> <a href="http://thecloudmarket.com/stats#/by_platform_definition" class="p-link--external">The Cloud Market</a>, August 2015 and <a href="http://superuser.openstack.org/articles/openstack-users-share-how-their-deployments-stack-up" class="p-link--external">OpenStack survey &mdash; May 2015</a></p>
     <p id="fn2" class="note"><a href="#r2">**</a> <a href="http://www.zdnet.com/article/ubuntu-linux-continues-to-rule-the-cloud/" class="p-link--external">zdnet.com</a>, 27 August 2015</p>
+    <p id="fn3" class="note"><a title="Jump to The Canonical Distribution of Kubernetes" href="#r3">***</a> Kubernetes&reg; is a registered trademark of The Linux Foundation in the United States and other countries, and is used pursuant to a license from The Linux Foundation</p>
```

Comment:
Do we not want the symbol in a `<sup>` tag?

Reply:
Done